partners.yml

```diff
@@ -0,0 +1,546 @@
+          format: international or swedish format
+          example: "+46731234567"
+        isNotPep:
+          type: boolean
+          description: Indicate if the customer is politically exposed
+          example: true
+        economicDetails:
+          $ref: "#/components/schemas/economicDetails"
+        bankDetails:
+          $ref: "#/components/schemas/bankDetails"
+        employer:
+          $ref: "#/components/schemas/employer"
+    
+    economicDetails:
+      type: object
+      required:
+      - monthlyIncome
+      - monthlyHousing
+      - monthlyLoans
+      - occupation
+      - kids
+      properties:
+        monthlyIncome:
+          type: number
+          description: Monthly income of the applicant before taxes in SEK
+          example: 30000
+          minimum: 0
+        monthlyHousing:
+          type: number
+          description: Monthly cost related to housing in SEK (e.g. monthly rent, monthly housing fees and monthly cost of mortgage). In case the applicant shares the cost with someone else, only his part should be reported.
+          example: 8000
+          minimum: 0
+        monthlyLoans:
+          type: number
+          description: Monthly cost for repaying other loans in SEK (Does not include mortgage as it is already included in the housing cost)
+          example: 1500
+          minimum: 0
+        mortgageAmount:
+          type: number
+          description: Total amount of mortgage for that customer in SEK
+          example: 2500000
+          minimum: 0
+        occupation:
+          type: string
+          description: Type of employment of the applicant. The enum values will be provided to the partner.
+          enum:
+          - permanent
+          - temporarly
+          - unemployed
+          - ...
+        kids:
+          type: number
+          description: Number of children of the applicant under 18
+          example: 2
+          minimum: 0
+        homeType:
+          type: string
+          description: Type of accomodation of the applicant. The enum values will be provided to the partner.
+          enum:
+          - villa
+          - bostadsrätt
+          - hyresrätt
+          - ...
+    
+    bankDetails:
+      type: object
+      properties:
+        bank:
+          type: string
+          description: Name of customer's bank. The enum values will be provided to the partner.
+          example: SEB
+          enum:
+            - nordea
+            - seb
+            - ...
+        clearingNumber:
+          type: string
+          description: Clearing number of the customer's bank account. 4 or 5 digits depending of the bank (e.g 7428 or 86439)
+          pattern: /^\d{4,5}$/
+        accountNumber:
+          type: string
+          description: Customer's bank account number, excluding the clearing number
+          pattern: /^\d{7,10}$/
```

Comment:
This has been changed actually to `\d{5,12}`. We should update as well in the broker api docs.